de.renew.Renew.yml

```diff
@@ -0,0 +1,42 @@
+id: de.renew.Renew
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: loadrenew
+finish-args:
+  - "--env=PATH=/app/jre/bin:/usr/bin:/app/bin"
+  - "--share=network"
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--filesystem=host" 
```

Comment:
Explanation of permissions:

- network access is needed because it is possible to connect to a different instance of Renew and inspect a simulation on a different machine
- filesystem host access is needed to store and load net files.

Reply:
I'm asking about host filesystem access, where do you need to save? Does it support filechoosers/filechooser portals? etc.

Reply:
There are configuration and log files stored in the home directory (~/.renew/ and ~/.renew.properties and ~/renewlogs/).
Users can load net files from and save net files to wherever they like and there are multiple ways files can be loaded/saved (from command line, via a swing file chooser and more).
Additionally it is possible to load plugins from a directory of choice.
The application is written in Java and it seems there is no easy way to use filechooser portals with Java (at least I have not found anything and I must admit that I do not fully understand the concept). 
Renew is often used in universities where net drives or such things are more common.
I would not want to limit the app to access only certain directories such as the home directory.